### PR TITLE
Events 2022-11-23

### PR DESCRIPTION
**Pull request by andrewpollack**

### CODE REVIEW

File: draft/2022-11-23-this-week-in-rust.md [hunk 1]
```diff
@@ -232,12 +218,16 @@ Rusty Events between 2022-11-23 - 2022-12-21 🦀
     * [**Rust Munich 2022 / 3 - hybrid**](https://www.meetup.com/rust-munich/events/289065390/)
 * 2022-12-01 | Virtual (Charlottesville, VA, US) | [Charlottesville Rust Meetup](https://www.meetup.com/charlottesville-rust-meetup/)
     * [**Exploring USB with Rust**](https://www.meetup.com/charlottesville-rust-meetup/events/289563986/)
+* 2022-12-01 | Virtual (Lehi, UT, US) | [Utah Rust](https://www.meetup.com/utah-rust/)
+    * [**Beginner Projects and Shop Talk with Food!**](https://www.meetup.com/utah-rust/events/289899804/)
```
Comment (mariannegoldin) [inline draft/2022-11-23-this-week-in-rust.md:222]: Consider adding for 12/1: https://www.meetup.com/microsoft-reactor-redmond/events/289830539/

File: draft/2022-11-23-this-week-in-rust.md [hunk 2]
```diff
@@ -250,18 +240,13 @@ Rusty Events between 2022-11-23 - 2022-12-21 🦀
     * [**Meetup: 25u16**](https://www.meetup.com/rust-saar/events/289663288/)
 * 2022-12-14 | Virtual (Boulder, CO, US) | [Boulder Elixir and Rust](https://www.meetup.com/boulder-elixir-rust/)
     * [**Monthly Meetup**](https://www.meetup.com/boulder-elixir-rust/events/zvxcsrydcqbsb/)
-
-### Asia
-
-* 2022-11-22 | Tokyo, JP | [Tokyo Rust Meetup](https://www.meetup.com/tokyo-rust-meetup)
-    * [**Fullstack Web Dev in Rust**](https://www.meetup.com/tokyo-rust-meetup/events/289661246/)
+* 2022-12-20 | Virtual (Washington, DC, US) | [Rust DC](https://www.meetup.com/rustdc//)
```
Comment (mariannegoldin) [inline draft/2022-11-23-this-week-in-rust.md:243]: Consider for 12/20: https://berline.rs/2022/12/20/rust-hack-and-learn.html

**Review by mariannegoldin — APPROVED** [review]
Looks great! Added a couple suggestions, approved.

File: draft/2022-11-23-this-week-in-rust.md (hunk 1 above)
Comment (andrewpollack) [reply draft/2022-11-23-this-week-in-rust.md:222]: Added!

File: draft/2022-11-23-this-week-in-rust.md (hunk 2 above)
Comment (andrewpollack) [reply draft/2022-11-23-this-week-in-rust.md]: Added!

**Pull request merged**